liquidator/liquidator.js

```diff
@@ -26,8 +40,22 @@ class Liquidator {
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
   queryAndLiquidate = async priceFunction => {
+    const { toBN, fromWei } = this.web3.utils;
+
     const contractTime = this.empClient.getLastUpdateTime();
-    const priceFeed = priceFunction(contractTime);
+    let priceFeed = priceFunction(contractTime);
+
+    // The `priceFeed` is a Number that is used to determine if a position is liquidatable. The higher the
+    // `priceFeed` value, the more collateral that the position is required to have to be correctly collateralized.
+    // Therefore, we add a buffer by reducing `priceFeed` to (`crThreshold` * `priceFeed`).
+    priceFeed = fromWei(toBN(priceFeed).mul(toBN(this.crThreshold)));
```

Comment:
I totally see what the math is doing here, but I'm having trouble intuiting precisely what 0.99 would imply here. Is there an intuitive explanation of crThreshold? I'm *guessing* that it would be incorrect to say that in the case of a crThreshold of 0.99 that the collateralization would need to be exactly 1% below the requirement before being liquidated? Is that the case? I'd like to have a 1 sentence explanation on how to figure out when liquidations will happen given a crThreshold value.

Reply:
`crThreshold`: If collateral falls to `crThreshold` percentage of the min collateral requirement, then it will be liquidated. For example: If the min. CR is %120 and the TRV is 100, then the minimum collateral requirement is 120, or 117.6 with the `crThreshold`.

Reply:
Does that language make it clear?

Reply:
Yep, that makes sense. I think I have a better intuitive understanding now.

I think the thing that's throwing me off a little is that as a user, I would expect to specify 0.01 to put a 1% threshold rather than 0.99. The explanation in that case would be that once the collateralization ratio gets more than x% below the requirement, the bot will liquidate. That may just be me, though. Obviously, it's easy to convert between the two in the code, so it's really just a question of what would be more natural for the user.

Reply:
>Obviously, it's easy to convert between the two in the code, so it's really just a question of what would be more natural for the user.

This is exactly why I chose to do it this way. I'll keep it this way and just revise the comment as above.

Reply:
Yeah I wanted to avoid the extra `.sub` operation which is why I did this but I do think its more intuitive to specify 2% instead of 98%. I'll make that change